index.js

```diff
@@ -16,18 +22,7 @@ module.exports = () => {
 		return pify(childProcess.execFile)(path.join(__dirname, 'lib', 'empty-recycle-bin.exe'));
 	}
 
-	return xdgTrashdir().then(dir => {
-		const paths = [
-			path.join(dir, 'files'),
-			path.join(dir, 'info')
-		];
-
-		return Promise.all(paths.map(pth =>
-			pify(fs.readdir)(pth).then(files =>
-				Promise.all(files.map(file =>
-					pify(rimraf)(path.join(pth, file)))
-				)
-			)
-		));
-	});
+	return xdgTrashdir.all()
+		.then(dirs => dirs.filter(pathExists.sync))
```

Comment:
no, but `emptyTrash` function first finds all files in that location (it doesn't remove the `Trash` folder itself, only its contents), so `fs.readdir` on a non-existing folder would fail.